### ISSUE TITLE
notification items have too small click areas

### ISSUE THREAD

**Issue by Simon-Laux** [pixelfed/pixelfed-rn#402]
I think the whole item should be the link, it is unintuitive that you can click different parts of it for different actions (go to profile, go to post)

also, maybe offtopic, when a mention is from a comment clicking on it does not lead me to the comment, just to the post the comment was made on

**Comment by Skivling** [comment]
I agree
